### DISCUSSION TITLE
Compatibility Request

### DISCUSSION THREAD

**Discussion by CommanderCrispy** [vosen/ZLUDA#68] [Ideas]
I want to ask for zluda compatibility with bricklink's stud.io, and more specifically, it's eyesight renderer to be added/worked on. It's been stuck in a "team green CUDA or super slow CPU rendering" state for forever, and it doesn't look like it'll ever change due to completely unmotivated devs. This is a spot where zluda could **_shine_**, but it doesn't look like it's able to implement itself properly at the moment.

**Comment by vosen** [comment]
I've just checked, ZLUDA can't even start trying to run it because it's not compiled with PTX assembly.
You'd need to start by convincing app creators to ship app with PTX assembly compiled
